src/Spectre.libGenetic/CrossoverOperator.cpp

```diff
@@ -24,10 +24,11 @@ limitations under the License.
 
 namespace spectre::algorithm::genetic
 {
-CrossoverOperator::CrossoverOperator(Seed rngSeed, size_t minimalFillup, size_t maximalFillup):
+CrossoverOperator::CrossoverOperator(Seed rngSeed, size_t minimalFillup, size_t maximalFillup, BaseIndividualFeasibilityCondition* individualFeasibilityCondition):
     m_RandomNumberGenerator(rngSeed),
     m_MinimalFillup(minimalFillup),
```

Comment:
I suppose that these `minimalFillup` and `maximalFillup` gets removed in next review? This functionality should be realized through `FeasibilityCondition`.

Reply:
I will be, yes, wanted short PR